Undefined symbols _kshadowobjref for ThreadLocal companion object in MPP used as library

Konan: `0.8-dev-2701`

Inside library tests, it works fine, but when used the library with `ThreadLocal` in another library, some symbols couldn't be found. Used `expected annotation class` for being able to use it in MPP:

https://github.com/korlibs/korlibs/commit/569b10d600d7aa06c287f096703599ab1ec915dd

```
Undefined symbols for architecture x86_64:
  "_kshadowobjref:com.soywiz.klock.DateTime.Companion", referenced from:
      ___unnamed_363 in combined.o
      ___unnamed_389 in combined.o
      ___unnamed_607 in combined.o
  "_kshadowobjref:com.soywiz.klogger.Logger.Companion", referenced from:
      ___unnamed_47 in combined.o
      _kfun:com.soywiz.korio.async.ExceptionHook.hook(#GENERIC_kotlin.Throwable)Generic in combined.o
      _kfun:com.soywiz.korio.async.Promise.complete$korio-native(#GENERIC;kotlin.Throwable?)com.soywiz.korio.async.Promise<#GENERIC> in combined.o
      ___unnamed_301 in combined.o
      ___unnamed_319 in combined.o
      ___unnamed_321 in combined.o
      ___unnamed_655 in combined.o
      ...
ld: symbol(s) not found for architecture x86_64
```

Reproduced this by travis here: https://travis-ci.org/korlibs/korlibs/builds/400568925#L6482

--- a/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt
+++ b/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt
@@ -38,7 +38,6 @@ import org.jetbrains.kotlin.ir.symbols.*
 import org.jetbrains.kotlin.ir.types.*
 import org.jetbrains.kotlin.ir.util.defaultType
 import org.jetbrains.kotlin.ir.util.getArguments
-import org.jetbrains.kotlin.ir.util.hasAnnotation
 import org.jetbrains.kotlin.ir.visitors.IrElementVisitorVoid
 import org.jetbrains.kotlin.ir.visitors.acceptChildrenVoid
 import org.jetbrains.kotlin.ir.visitors.acceptVoid
@@ -50,7 +49,7 @@ import org.jetbrains.kotlin.resolve.descriptorUtil.fqNameUnsafe
 private val threadLocalAnnotationFqName = FqName("konan.ThreadLocal")
 
 val IrClassSymbol.objectIsShared get() =
-    !owner.hasAnnotation(threadLocalAnnotationFqName)
+    !descriptor.annotations.hasAnnotation(threadLocalAnnotationFqName)
 
 internal fun emitLLVM(context: Context) {
     val irModule = context.irModule!!

